tests/Feature/DeleteUserTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DeleteUserTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_if_you_can_delete_the_user()
+    {
+        $this->artisan('passport:install', ['--force' => true]);
+        $createdUsers = User::factory()->count(10)->create();
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('token')->nullable();
+        });
+        foreach ($createdUsers as &$createdUser) {
```

Comment:
You can create 1 user and do `Passport::actingAs($user)` before request. It will allow you to not add Authorization header.